holidayextras/plugin-elephant
# NB-1314 updated elephant

#### What does this PR do? (please provide any background)
[NB-1314](https://hxshortbreaks.atlassian.net/browse/NB-1314)
[NB-1600](https://hxshortbreaks.atlassian.net/browse/NB-1600)
Updated Elephant

#### What tests does this PR have?
None

#### How can this be tested?
Make a booking through The Works, then once the Elephant orders' table has been updated with the new column (referrerGroup), you will be able to see the booking with an referrerGroup (encoreTickets).
Also, the data platform event with have the referrerGroup.

#### Any tech debt?
None

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![]()

- I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
